apps/mobile/src/components/Notification/NotificationSkeleton.tsx

```diff
@@ -26,6 +27,7 @@ type Props = PropsWithChildren<{
   responsibleUserRefs: NotificationSkeletonResponsibleUsersFragment$key;
   shouldShowFollowBackButton?: boolean;
   overridePfpElement?: React.ReactNode;
+  hasAdmired?: boolean;
```

Comment:
could we rename this to be slightly more explicit? `hasReceivedAdmire`

Reply:
good call! something like `isAdmireNotificationType` might even be more explicit, since it looks like it denotes that it's an admire notification

which makes me wonder if it needs to be a prop at all since we have access to the notification typename, which we can use to determine if it's an admire notification

like
isAdmireNotificationType = useMemo(()=>['SomeoneAdmiredYourCommentNotification', 'otherAdmireNotificationTypename'].includes(notification.__typename),[])

Reply:
That smart! Updated to remove the props